tahrir/templates/user.mak

```diff
@@ -204,11 +204,24 @@
             (<strong>${"{0:.1f}".format(percent_earned)}%</strong> of total).
           % endif
         </div>
+        % for tag in request.registry.settings.get('tahrir.display_tags', '').split(","):
+        % if tag in sum([badge.tags.strip(",").split(",") for badge in user_badges], []):
+        <h3 class="section-header">${tag.title()} Badges</h3>
         <div class="flex-container">
-          % for i, badge in enumerate(user_badges):
+          % for badge in [b  for b in user_badges if tag in b.tags]:
```

Comment:
Is ``b.tags`` here a list or a string? If it is a string that might cause problem if we have: event and non-event tags